src/components/error-boundary/Fallback.tsx

```diff
@@ -25,10 +25,20 @@ const Message = styled(View)({
   textAlign: 'center',
 });
 
-export default function Fallback() {
-  const { colors } = useTheme();
```

Comment:
why remove `useTheme` here?

Reply:
Having a hook inside of the `Sentry.ErrorBoundary` fallback prop wasn't allowed for whatever reason. I think it needs to be PureComponent